src/streaming/text/TextController.js

```diff
@@ -166,6 +179,36 @@ function TextController(config) {
         textSourceBuffers[streamId].addEmbeddedTrack(mediaInfo);
     }
 
+    /**
+     * Event that is triggered if a font download of a font described in an essential property descriptor
+     * tag fails. 
+     * @param {FontInfo} font - font information
+     * @private
+     */
+    function _onFontDownloadFailure(font) {
+        logger.error(`Could not download ${font.isEssential ? 'an essential' : 'a'} font - fontFamily: ${font.fontFamily}, url: ${font.url}`);
+        if (font.isEssential) {
+            let idx = textTracks[font.streamId].getTrackIdxForId(font.trackId);
+            textTracks[font.streamId].setModeForTrackIdx(idx, Constants.TEXT_DISABLED);
```

Comment:
I wonder if this behavior is completely spec conform. The spec states:

> If the font download was signalled using the EssentialProperty descriptor, the Adaptation Set containing the descriptor shall not be presented.

While the track is disabled by default if not available and an `EssentialProperty` was used the user can still select it manually by turning subtitles off and on. This is a minor issue, but it might be cleaner to blacklist the AdaptationSet in this case.

Reply:
Sorry missed your comment above. I suggest we remove disabled tracks from the controlbar

Reply:
No problem. I'll look at the controlbar code.

Reply:
I am seeing an error thrown by the Controlbar when playing one of our other teststreams like https://livesim2.dashif.org/vod/testpic_2s/multi_subs.mpd

````
ControlBar.js:604 Uncaught TypeError: Cannot read properties of undefined (reading 'toString')
    at _getNativeVideoTrackMode (ControlBar.js:604:44)
    at contentFunc (ControlBar.js:640:27)
    at ControlBar.js:766:26
    at Array.forEach (<anonymous>)
    at getMenuContent (ControlBar.js:765:13)
    at createMenu (ControlBar.js:700:44)
    at createCaptionSwitchMenu (ControlBar.js:644:27)
    at TextTrackList._onTracksChanged (ControlBar.js:659:9)
````